apps/crn-frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -54,6 +54,7 @@ const DuplicateOutput: FC = () => {
           link: undefined,
           title: `Copy of ${output.title}`,
         }}
+        descriptionUnchangedWarning
```

Comment:
Hey, I wonder if we could rename this `descriptionUnchangedWarning` to something like `isDuplicate` or `originatedFromDuplicate` :)

When I saw this truthy boolean I at first I was asking myself if this was ever recalculated to see if the description changed then only after I understood that this `descriptionUnchangedWarning` was only to flag that this comes from a duplicate

Reply:
I feel that its better to describe how the component behaves with the prop than expect the next programmer to have the domain knowledge of thats how a duplicate behaves